sass/cd/cd-header/_cd-user-menu.scss

```diff
@@ -67,6 +67,58 @@
   }
 }
 
+// Placeholder menu item with + icon, based on placement.
+.logged-in .cd-user-menu > li:first-child {
+  a {
+    padding: 0 12px 0 26px;
+
+    @include tablet {
+      padding: 0 20px 0 34px;
+    }
+  }
+  &:before {
+    position: absolute;
+    content: "";
+    height: $cd-global-header-height;
+    width: 14px;
+    left: 7px;
+    background-image: url(../img/icons/plus.svg);
+    background-repeat: no-repeat;
+    background-position: 0 46%;
+    background-size: 14px 14px;
+
+    @include tablet {
+      left: 11px;
+    }
+  }
+}
+
+// Log in / Log out menu item, based on placement.
+.cd-user-menu > li:last-child {
```

Comment:
Applying the icons based on position within menu seems risky. Even if you or I set the menu up properly someone can go rearrange a menu without any fanfare and then the icon applies to some other link. Aside from adding yet another module like https://www.drupal.org/project/menu_attributes I'm not sure how to control this with 100% certainty.

Then again this can easily be considered an edge case and we can simply file a ticket to address the situation when it actually occurs.

Reply:
I agree, it's hard to know how to best control this. I think it's ok to assume the login/logout is last (but I take your point about content editors rearranging). This gets even worse if there are more than 2 items (:nth-child etc will make things even more brittle).
Based on what we know now about the specific implementations, I think we can treat it as an edge case. Let's see how/when it breaks and adjust accordingly.